python/ctsm/toolchain/gen_mksurfdata_namelist.py

```diff
@@ -463,16 +463,16 @@ def check_ssp_years(start_year, end_year):
     """
     Check years associated with SSP period
     """
-    if int(start_year) > 2015:
+    if int(start_year) > 2022:
```

Comment:
Change to 2023 here and throughout this PR, when TRENDY2024 becomes available.

Similarly 2023 to 2024 where necessary.

Reply:
Eventually also change the new filepaths to lnd/clm2/rawdata/...

Reply:
The former is done. The latter comes up in Erik's review below and in the unchecked checkboxes above, so I'm resolving this one.

Reply:
Oh, I see you already had a reminder about moving to input data. Didn't see that until after I already said it...

Reply:
I don't mind, Erik. I appreciate having your confirmation.